src/layout.c

```diff
@@ -351,6 +351,13 @@ void handle_success(void) {
 		}
 	}
 
+	if (!head_changing_adaptive_sync && cfg->change_success_cmd) {
```

Comment:
We can enhance these suppressions in subseqent PR.

Reply:
Sure!